docs/src/components/layouts/utility-page-layout/utility-class-tables/color-demonstration.jsx

```diff
@@ -0,0 +1,96 @@
+import React from 'react';
+import PropType from 'prop-types';
+import UtilityClassTitleBar from '../utility-class-example-title-bar';
+
+/**
+ * @typedef ColorDemonstrationClass
+ * @param {string} class_name The class name.
+ * @param {string} hex_value The calculated value of the size of the token.
+ * @param {string} additional_example_classes Any additional classes to add to the example.
+
+ */
+
+/**
+ * Helper to draw a utility class example.
+ * @param {ColorDemonstrationClass} utilityClass The utility class.
+ * @returns
+ */
+const drawRow = (utilityClass, utilityClassDisplayParams) => {
+	// Merge example classes.
+	const classNames = [
+		utilityClassDisplayParams.additional_example_classes,
+		utilityClass.additional_example_classes,
+		utilityClass.class_name,
+	].join(' ');
+
+	const backgroundColor = utilityClass.class_name?.includes('bg') || false;
+	const text_color_class = !backgroundColor
+		? utilityClass.class_name?.replace('text', 'bg')
+		: utilityClass.class_name;
+	const isSubHeader = Boolean(utilityClass.sub_header);
+	return (
+		<div
+			key={utilityClass.class_name}
+			className="grid-row site-utility-examples__row">
+			{isSubHeader ? (
+				<div className="site-utility-examples__sub-header">
+					{utilityClass.sub_header}
+				</div>
+			) : (
+				<>
+					{backgroundColor && (
+						<div
+							className={`${utilityClass.class_name} grid-col-1 site-utility-examples__bg-color-chip`}></div>
+					)}
+					<div
+						className={`grid-col site-utility-examples__class${
```

Comment:
I still don't see why the background information cannot be a modifier or something of `__class`. Really it feels like this is changing because the type of display.

Reply:
If this background color is just modified in the `__class` CSS, it is overridden by the background color being displayed in the color chip.